pennylane/ops/snapshot.py

```diff
@@ -61,7 +61,7 @@ def circuit():
     num_params = 0
     grad_method = None
 
-    def __init__(self, tag=None, do_queue=True):
+    def __init__(self, tag=None, do_queue=None):
```

Comment:
Could add `do_queue` and `id` to the "Args" section above.

Reply:
94915cd8a1748832d3d6d4ef3b35423e7b8ecb0b